OrbitCore/LinuxUtils.cpp

```diff
@@ -96,6 +96,7 @@ void ListModules(pid_t pid,
   struct AddressRange {
     uint64_t start_address;
     uint64_t end_address;
+    bool is_kernel_module;
```

Comment:
This should not be a boolean, but rather an enum ModuleType (or something equivalent).